params.yaml

```diff
@@ -294,27 +287,27 @@ model:
       # for setting this parameter is to discover a good fixed value using CV,
       # then manually set that value for non-CV runs (which don't use
       # early stopping by default)
-      num_iterations: 2500
-      learning_rate: 0.015
+      num_iterations: 2275
+      learning_rate: 0.011
```

Comment:
These params are pulled from an earlier long CV run (`2024-02-11-gracious-manasi`) that yielded similar, but _slightly_ improved results.